### ISSUE TITLE
A11Y Lab - Page Titled

### ISSUE THREAD

**Issue by glenn-sorrentino** [scidsg/hushline#153]
The intent of this Success Criterion is to help users find content and orient themselves
within it by ensuring that each Web page has a descriptive title. Titles identify the
current location without requiring users to read or interpret page content.
Level Compliance 2.4.2.
A No
23
• The page <title> must be accurate and informative.
• The <title> should be concise.
• The page <title> should be unique, if possible.
• Unique information should come first in the <title>.
Currently, HushLine website home and try.HushLine have the same title:

**Comment by glenn-sorrentino** [comment]
👍 Fixed

**Comment by glenn-sorrentino** [comment]
https://github.com/scidsg/project-info/blob/main/hush-line/1.%20Accessibility/WCAG%202.1%20AAA_HushLine%20website.pdf
